app/views/users/_import_account_modal.haml

```diff
@@ -0,0 +1,57 @@
+.modal.fade{id: "importAccountModal",
+            tabindex: "-1",
+            role: "dialog",
+            aria: {labelledby: "importAccountModalLabel", hidden: "true"}}
+  .modal-dialog
+    .modal-content
+      .modal-header
+        %button.close{type: "button", data: {dismiss: "modal"}, aria: {hidden: "true"}}
+          &times;
+        %h3.modal-title{id: "importAccountModalLabel"}
+          = t("users.import_modal.title")
+        .modal-body
+          %p
+            = simple_format(t("users.import_modal.subtext", {accountname: current_user.diaspora_handle}))
+          %ol
+            %li
+              = t("users.import_modal.list1")
+            %li
+              = t("users.import_modal.list2")
+            %li
+              = t("users.import_modal.list3")
+            %li
+              = t("users.import_modal.list4")
+          %strong
+            = simple_format(t("users.import_modal.closing_text"))
+          = form_for current_user,
+                     url: edit_user_path,
+                     html: {method: :put, multipart: true, class: "form-horizontal"} do |f|
+            .row
+              .col-md-12
+                .upload-btn-wrapper
+                  .btn.btn-primary
+                    Select profile archive
+                  = f.file_field :export, accept: "application/json, application/zip, application/gzip",
+                    id: 'profile-file-btn'
+                  %span#profile-file-chosen
+                    No profile set
+            .row
+              .col-md-12
+                .upload-btn-wrapper
+                  .btn.btn-primary
+                    Select photo archive
+                  = f.file_field :exported_photos_file, accept: "application/zip",
```

Comment:
- Layout/SpaceAfterColon: Space missing after colon.
- Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.